tests/Feature/Consumables/Api/ConsumableFileTest.php

```diff
@@ -103,7 +103,7 @@ public function testConsumableApiDownloadsFile()
         // Upload a file
         $this->actingAsForApi($user)
             ->post(
-                route('api.files.store', ['object_type' => 'consumables', 'id' => $consumable->id]), [
+                route('api.files.store', ['object_type' => 'consumables', 'id' => $consumable->id, 'order' => 'asc']), [
```

Comment:
I think this change should be on line 119.

Reply:
I did, yes :)